pkg/master/master.go

```diff
@@ -586,13 +594,26 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
-func (m *Master) NewBootstrapController() *Controller {
+// If endpointReconcilerConfig.Interval is 0, the default value of 10 seconds will be used instead.
+// If endpointReconcilerConfig.Reconciler is nil, the default MasterCountEndpointReconciler will be
+// used.
+func (m *Master) NewBootstrapController(endpointReconcilerConfig EndpointReconcilerConfig) *Controller {
+	if endpointReconcilerConfig.Interval == 0 {
+		endpointReconcilerConfig.Interval = 10 * time.Second
```

Comment:
i just think these are hidden, so const was appealing  especially if the godoc says "10 seconds" it could just say the value in this const instead and leave it to the reader.